pyscriptjs/src/components/base.ts

```diff
@@ -130,21 +130,19 @@ export class BaseEvalElement extends HTMLElement {
         try {
             source = this.source ? await this.getSourceFromFile(this.source)
                                  : this.getSourceFromElement();
+            const is_async = source.includes('asyncio')
 
             await this._register_esm(pyodide);
-
-            if (source.includes('asyncio')) {
+            if (is_async) {
```

Comment:
this change is fine but honestly I think we should just remove and always run async. There's not fundamental diff on the pyodide side... (saying out loud just to share :) )

Reply:
I fully agree, I actually thought we had already done that and it got reverted for some reason.